Apps/WebClient/src/ClientApp/src/components/calendar/calendar.vue

```diff
@@ -77,11 +78,15 @@ export default class CalendarComponent extends Vue {
     weekNames!: Array<string>;
 
     private availableMonths: Date[] = [];
-
     private currentDate: Date = new Date();
+    private eventBus = EventBus;
 
     private mounted() {
         this.updateAvailableMonths();
+        var self = this;
+        this.eventBus.$on("timelinePageUpdate", function (eventDate: Date) {
```

Comment:
Should we have a const for this timelinePageUpdate as we have to use it at least in 2 different places?

Reply:
I think we can. I was waiting for your PR to change it since that one has the constants file. In any case, we should open a task for this to comb over the bus utilization and change it throughout